src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -115,6 +115,14 @@ TEST_F(Cgroups2Test, ROOT_CGROUPS2_AssignProcessToCgroup)
   AWAIT_EXPECT_WTERMSIG_EQ(SIGKILL, process::reap(pid));
 }
 
+
+TEST_F(Cgroups2Test, CGROUPS2_Path)
+{
+  EXPECT_EQ("/sys/fs/cgroup/", cgroups2::path(cgroups2::ROOT_CGROUP));
```

Comment:
I guess this is indicating that you have two ways of calling this function?

1. Pass a "cgroup" relative to the hierarchy, you get back the absolute path.
2. Pass a "cgroup" using an absolute path, the absolute path must include the hierarchy already in this case, e.g. if you pass "/foo", you'll only get back "/foo", not "/sys/fs/cgroup/foo".